_posts/en/newsletters/2025-06-06-newsletter.md

```diff
@@ -0,0 +1,159 @@
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Syncing full nodes without witnesses:** Jose SK [posted][sk nowit]
+  to Delving Bitcoin a summary of a [analysis][sk nowit gist] he
+  performed about the security tradeoffs of allowing newly started full
+  nodes with a particular configuration to avoid downloading some
+  historic blockchain data.  By default, Bitcoin Core nodes use the
+  `assumevalid` configuration setting that skips validation of scripts
+  in blocks created more than a month or two before the release of the
+  version of Bitcoin Core being run.  Although disabled by default, many
+  users of Bitcoin Core also set a `prune` configuration setting that
+  deletes blocks some time after validating them (how long blocks are
+  kept depends on the size of the blocks and specific setting selected
+  by the user).
+
+  SK argues that witness data, which is only used for validating
+  scripts, should not be downloaded by pruned nodes for assumevalid
+  blocks because they won't use it for validating scripts and will
+  eventually delete it.  Skipping download of witnesses "can cut
+  bandwidth usage by over 40%," he writes.
+
+  Ruben Somsen [argues][somsen nowit] that this does change the security
+  model, at least some.  Although scripts aren't validated, the
+  downloaded data is validated against the commitment from the block
+  header merkle root to the coinbase transaction to the witness data.
+  This ensures the data was available and uncorrupted at the time the
+  node was initially synced.  If nobody routinely validates the
+  existence of the data, it could conceivably be lost, as [has
+  happened][ripple loss] to at least one altcoin.
+
+  The discussion was ongoing at the time of writing.
+
+## Changing consensus
+
+_A monthly section summarizing proposals and discussion about changing
+Bitcoin's consensus rules._
+
+- **Quantum computing report:** Clara Shikhelman [posted][shikelman
+  quantum] to Delving Bitcoin the summary of a [report][sm report] she
+  co-authored with Anthony Milton about the risks to Bitcoin user of
+  fast quantum computers, an overview of several pathways to [quantum
+  resistance][topic quantum resistance], and an analysis of tradeoffs
+  involved in upgrading the Bitcoin protocol.  The authors find 4 to 10
+  million BTC are potentially vulnerable to quantum theft, some
+  mitigation now is possible, Bitcoin mining is unlikely to be
+  threatened by quantum computing in the short or medium term, and
+  upgrading requires widespread agreement.
+
+- **Transaction weight limit with exception to prevent confiscation:**
+  Vojtěch Strnad [posted][strnad limit] to Delving Bitcoin to propose
+  the idea for a consensus change to limit the maximum weight of most
+  transactions in a block.  The simple rule would only allow one
+  transaction per block (besides the coinbase transaction) to exceed
+  400,000 weight units (100,000 vbytes).  Strnad and others described
```

Comment:
This reads like there can be more transactions in the block along with the oversized one. Crucially, the rule would forbid such a block from having any other transactions.

Reply:
I didn't realize that, sorry!  Will revise.